app/client/components/cancel/cancellationFlow.tsx

```diff
@@ -169,27 +155,11 @@ export interface RouteableStepPropsWithCancellationFlow
 export const CancellationFlow = (
   props: RouteableStepPropsWithCancellationFlow
 ) => (
-  <div>
-    <PageContainer>
-      <h1 css={{ fontSize: "24px" }}>
-        Cancel your{" "}
-        {props.productType.includeGuardianInTitles ? "Guardian " : ""}
```

Comment:
in JSX when mixing text and {variables} the spaces can be lost so the `{" "}` forces a space

in this particular case yes I could've omitted the `{" "}` and had the ternary as `{props.productType.includeGuardianInTitles ? " Guardian " : " "}` however it would be inconsistent so perhaps not worth changing it now